docs/api.md

```diff
@@ -84,7 +84,7 @@ following API groups:
 API Group | Status | Details
 --------  | ------ | -------
 [`azure.crossplane.io/v1alpha3`] | Alpha | Core Azure resources.
-[`cache.azure.crossplane.io/v1alpha3`] | Alpha | Azure Redis managed resources and classes.
+[`cache.azure.crossplane.io/v1beta1`] | Beta | Azure Redis managed resources and classes.
```

Comment:
This link will be broken until the API docs are regenerated. Do you think we should go ahead and regenerate the docs for just Azure Redis here, or just wait until the next release and regenerate for all updated resources?

Reply:
Probably worth regenerating the API docs as part of this PR. https://github.com/negz/crossdocs/pull/1 shows how @jbw976 bumped it for 0.4. It's probably much less work for this change - I'm guessing just replacing cache v1alpha3 with v1beta and running crossdocs.

Reply:
lemme know if you run into any issues @hasheddan, i should still remember most of my recent experience in that crossdocs tool 🤔

Reply:
Done!